### ISSUE TITLE
karibu-testing-v10:1.1.14 - MockVaadin.setup fails with Java 13

### ISSUE THREAD

**Issue by tmohme-hypoport** [mvysny/karibu-testing#29]
With karibu-testing-v10:1.1.14, Vaadin 14.0.7 and Java 13 we _always_ get the following Exception when trying to setup a test:

```
java.lang.NoSuchFieldException: modifiers

	at java.base/java.lang.Class.getDeclaredField(Class.java:2412)
	at com.github.mvysny.kaributesting.v10.UtilsKt.makeNotFinal(Utils.kt:61)
	at com.github.mvysny.kaributesting.v10.MockNpmTemplateParser$Companion.install(MockNpmTemplateParser.kt:129)
	at com.github.mvysny.kaributesting.v10.MockVaadin.mockVaadin14(MockVaadin.kt:152)
	at com.github.mvysny.kaributesting.v10.MockVaadin.setup(MockVaadin.kt:126)
	at com.github.mvysny.kaributesting.v10.MockVaadin.setup(MockVaadin.kt:106)
	...
```

To me it looks like in [Utils.kt#61](https://github.com/mvysny/karibu-testing/blob/c8cbef74b06846f847ec1d3afa9ef212975b3581/karibu-testing-v10/src/main/kotlin/com/github/mvysny/kaributesting/v10/Utils.kt#L61) you try to change the "modifiers" of class "java.lang.reflect.Field" to being accessible and not final instead of the static "INSTANCE" field in class "com.vaadin.flow.component.polymertemplate.NpmTemplateParser"?!

We verified that (at least) all karibu-testing-v10 versions after 1.1.11 show this behaviour.

**Comment by mvysny** [comment]
Unfortunately the NpmTemplateParser.INSTANCE field is final and cannot be modified without nasty reflection magic. There is more information at https://github.com/mvysny/karibu-testing/issues/26

I'm sad that the reflection magic doesn't work on Java 13; I'll add a more clear error message into Karibu-Testing. Meanwhile, please vote for https://github.com/vaadin/flow/issues/6537 to be fixed.

As a workaround:
1. Please use Java 11 - the "nasty reflection" solution works there.
2. Downgrade to Karibu-Testing 1.1.11. It works around NpmTemplateParser in a different way, but it may not work for you properly.

I'll add a test for Java 13 and show a more meaningful error message.

**Comment by mvysny** [comment]
The fix has been merged into Vaadin, but will only be present in Vaadin 15+. The fix is not yet present in Vaadin 15.0.0.beta1.

**Comment by mvysny** [comment]
Fixed - Karibu-Testing 1.2.x is now compatible with Java 13, but only works with Vaadin 15+.
